src/w2grid.js

```diff
@@ -7235,20 +7252,6 @@
                     (isRowSelected && this.selectType == 'row' ? ' w2ui-selected' : '') +
                     (record.w2ui && record.w2ui.editable === false ? ' w2ui-no-edit' : '') +
                     (record.w2ui && record.w2ui.expanded === true ? ' w2ui-expanded' : '') + '" ' +
-                (summary !== true ?
```

Comment:
Did you account for summary records in your changed code?

Reply:
the summary recodrs parent don't have an event handler